src/state/network/reducer.js

```diff
@@ -92,6 +94,13 @@ const reducer = (state = initialState, { type, payload }) => {
     case types.UPDATE_SCROLL_TO_INDEX: {
       return state.set('scrollToIndex', payload);
     }
+    case types.SELECT_REQUEST: {
+      return state.withMutations((newState) => {
+        newState
+          .set('selectedReqIndex', payload)
+          .set('reqDetail', payload || payload === 0 ? state.getIn(['data', payload]) : null);
```

Comment:
is reqDetail used anywhere or is this for a future pr?
also is the ternary necessary? if payload is undefined `state.getIn(['data', payload])` should also return undefined.

Reply:
Yeah, `reqDetail ` is related to future PR. I might wrap it in `Map`.